lib/confirmation.rb

```diff
@@ -0,0 +1,20 @@
+class Confirmation
+
+	attr_reader :checkout, :total
+
+	def initialize(checkout)
+		@checkout = checkout
+		@total = 0
+	end
+	
+	def user_cost (user_total)
+		total_cost
+		raise "The totals don't match, ours: £#{total} yours: £#{user_total}!" if total != user_total
+		"Your order: #{checkout}. Comes to a total of £#{@total}. Would you like to proceed?"
```

Comment:
Metrics/LineLength: Line is too long. [87/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)